doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -2483,11 +2484,14 @@ Fields
 * "cyu": List of found CYUs in the packet
 * "cyu[].hash": CYU hash
 * "cyu[].string": CYU string
+* "ja3": The JA3 fingerprint consisting of both a JA3 hash and a JA3 string
+* "ja3s": The JA3S fingerprint consisting of both a JA3 hash and a JA3 string
+* "ja4": The JA4 client fingerprint for QUIC
 
 Examples
 ~~~~~~~~
 
-Example of QUIC logging with a CYU hash:
+Example of QUIC logging with CYU, JA3 and JA4 hashes (artificial):
```

Comment:
What does artificial mean here ?

Reply:
I didn't run Suricata to produce these example data, but just produced the example from no real-life traffic. JA3 and JA4 are not correlated here but just chosen to illustrate the format.

If you want, I can clarify that in the documentation. I just wanted to avoid readers getting confused by these not being possible in reality.

Reply:
Done